.zuul.d/project-templates.yaml

```diff
@@ -19,6 +19,7 @@
               - name: github.com/ansible-collections/frr.frr
               - name: github.com/ansible-collections/ibm.qradar
               - name: github.com/ansible-collections/junipernetworks.junos
+              - name: github.com/ansible-collections/kubernetes.core
```

Comment:
We should actually include all the network collection. So the question is basically: Is it ok to use the same EE to test the Cloud collections.
As the security collections are already part of the list, I feel like it should be doable.

Reply:
I would say, we don't want to include all the network collections by default.   So far, we've been targeting content that redhat is maintaining. Which k8s falls into.

We've overloading the term network-ee which is fine for me. We likely should change this to something more generic, like zuul-ee / test-ee.  To identify it is used for testing thing.

Reply:
Great, thank you @pabelanger and @Akasurde!